scripts/r2s.py

```diff
@@ -187,6 +187,8 @@ def step2():
         tags = {('TOTAL', dt): tag_name}
         photon_source_hdf5_to_mesh(mesh, h5_file, tags, sub_voxel=sub_voxel,
                                    cell_mats=cell_mats)
+        p_src_filename = '{0}_{1}.h5m'.format(output, i+1)
+        mesh.write_hdf5(p_src_filename)
```

Comment:
Why do we write the file twice?

Reply:
The file should be written once. Fixed in https://github.com/pyne/pyne/pull/1355/commits/97cf34d17c17c0de7faedf2d229936635063b72d.